src/components/Material/ExportSideBar/@shared/HorizontalLayout/HorizontalLayout.styles.js

```diff
@@ -0,0 +1,20 @@
+export default (theme) => ({
+    root: {
+        display: 'inline-flex',
+        alignItems: 'flex-end',
+
+        '& > div': {
+            flex: '1 1 auto',
+            display: 'grid',
+            marginLeft: '8px',
```

Comment:
je hoeft niet per se px aan te geven, dat is de default unit

```suggestion
            marginLeft: 8,
```